Makefile

```diff
@@ -153,9 +155,9 @@ CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
 	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.1)
 
-KUSTOMIZE = $(shell pwd)/bin/kustomize
+KUSTOMIZE = $(GOBIN)/kustomize
```

Comment:
Any reason to change this? It's not consistent with 'CONTROLLER_GEN', 'ENVTEST'. They are using '$(shell pwd)/bin/'.

Reply:
Change to previous definition